ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.4.yaml

```diff
@@ -0,0 +1,57 @@
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
```

Comment:
This should be golang 1.22 for all of the oadp-1.4 builds.

Reply:
Okay, now this is using `rhel-9-golang-1.22-openshift-4.17`